src/Transport/Addressing/Partitioning/ICacheableNamespacePartitioningStrategy.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace NServiceBus.Transport.AzureServiceBus
+{
+    public interface ICacheableNamespacePartitioningStrategy
```

Comment:
For better compatibility this should probably implement `INamespacePartitioningStrategy`

Reply:
> For better compatibility this should probably implement INamespacePartitioningStrategy

@danielmarbach can you elaborate? Compatibility with what?
In ASB v8 we'll deprecate `ICacheableNamespacePartitioningStrategy`, and `INamespacePartitioningStrategy` will have the `SendingNamespacesCanBeCached` property. Therefore if you've implemented `ICacheableNamespacePartitioningStrategy` for a custom strategy in v7, your strategy will not undergo any change other than dropping ``ICacheableNamespacePartitioningStrategy`. Am I missing something? Thanks.

Reply:
Then let's at least rename `ICacheableNamespacePartitioningStrategy` to something like `ICacheSendingNamespaces` or similar. Because it is cleary not a strategy